pkg/utils/webhook/add.go

```diff
@@ -0,0 +1,56 @@
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package webhook
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+)
+
+// ServerConfig is a completed webhook server configuration.
+type ServerConfig struct {
+	// Name is the webhook server name
+	Name string
+	// Mode is the webhook client config mode (service or url).
```

Comment:
do you know when we need url mode?
Is it for local testing?

Reply:
I copy-pasted that from our Gardener extension webhook without thinking much about it. Not sure, if we need it.

Reply:
URL mode seems not necessary and has been removed.